Paola issues

paolap opened the pull request:
This is addressing a few issues:

#71 making sure we have correspondence between indexes of concepts and tech pages
#92 there was an empty dropdown in this page, I fixed that by adding the tar cheat sheet info
#90 removed a pin to an older version of Jupyter-book
#83 I covered the example mentioned 
#73 added an introduction to this new section in the introduction page (please review)
And #53 , this has become a bit bigger, I reviewed the entire publishing section (opened new issues) and:
 1) fixed an issue with tabs that weren't showing correctly in publishing options
2)  created a new page publishing procedure to cover the generalised case before going into the options and the details of specific options. This isn't quite complete (see missing abstract for example) but I would like some feedback before proceeding.
It also might help to review/merge to at least fix the other issues.

I'm trying to encourage our users to rely on it now they need to manage their data on their own

hot007 commented on Governance/publish/publish-procedure.md:
```diff
@@ -0,0 +1,97 @@
+
+**Authors and collaborators**
+
+Another important choice is the authors and collaborators. As for a paper only people who contributed to the data (the data not the research project itself) should be listed as authors and where appropriate people who helped for example with the publishing process itself, formatting the data etc as collaborators. See the [authorship page](../concepts/authorship) in concepts.
```
```suggestion
Like journal manuscripts, datasets (and software) have an authorship list and collaborators. As for a paper, only people who contributed to the dataset (the data not the research project itself) should be listed as authors and where appropriate people who helped for example with the publishing process itself, formatting the data etc as collaborators. See the [authorship page](../concepts/authorship) in concepts.
```

hot007 reviewed (changes requested):
Um, I think this is the right button to press but apologies if not - I approve this PR pending a few minor corrections.

paolap commented on Governance/concepts/other-conventions.md:
````diff
@@ -29,3 +29,7 @@ IMOS] has very specific extensions of the CF conventions for different kind of o
 * [COMODO](https://web.archive.org/web/20160417032300/http://pycomodo.forge.imag.fr/norm.html) ??? still used?
+
+```{note} Potential clashes
+Some of these conventions are a spinoff of the CF Conventions and so there's an expectation when applied that the files will also be CF compliant. However, as conventions are ever-evolving documents and the groups working on specific conventions are different, it is possible for them to introduce requirements that clash with the CF conventions. An example of this is the different uses for cf_role ...
````
good spotting! Fixed

paolap commented on Governance/publish/publish-options.md:
````diff
@@ -1,27 +1,27 @@
 
-:::{tab-item} CSIRO 
+```{tab-item} CSIRO 
 Data, software and links to external data holdings (such as NCI) can be added to the [CSIRO Data Access Portal](https://data.csiro.au/) (DAP) by **CSIRO staff** only. More information on creating metadata records and uploading files to the CSIRO DAP can be found on the [DAP Help Guide](https://confluence.csiro.au/display/dap/Deposit+and+Manage+Data) (staff only access).
````
I think we can leave them both we need the first anyway to see what DAP is and it actually help also authors understanding how the data is accessed so it can't hurt. I've updated the links.

paolap commented on Governance/publish/publish-csiro-dap.md:
```diff
@@ -3,8 +3,3 @@
 Data, software and links to external data holdings (such as NCI) can be added to the [CSIRO Data Access Portal](https://data.csiro.au/) (DAP) by **CSIRO staff** only. More information on creating metadata records and uploading files to the CSIRO DAP can be found on the [DAP Help Guide](https://confluence.csiro.au/display/dap/Deposit+and+Manage+Data) (staff only access).
 
  CSIRO-affiliated data can be published in the DAP and the lead creator does not have to be CSIRO staff member.
-
```
I temporarily updated the links as for the other part just to be consistent

paolap replied:
there's a newly created issue #88

paolap commented on Governance/publish/publish-procedure.md:
```diff
@@ -0,0 +1,97 @@
+ 
+2. If the output is big publish only a subset. If methods are well described, the software used is easily available, then publishing only the subset of data that underlines a publication is sufficient. For example, the post-processed output is sufficient for a model simulation. However, the model version and configuration used, the input data and model source code should be documented.
+ 
+3. It's essential to consider an end user point of view. What would a user look for when considering using a dataset? What kind of information is essential for the data to be usable? Which additional information would make its use easier?
```
Good point, I was going to also add something like that in the abstract part which I haven't tackle yet, as giving context to the data helps highlighting that too, but it's worth to make it explicit

paolap replied:
Re-reading this I might move the current point 3 to the "abstract" I haven't wrote yet because it's something to consider in the data description and make your suggestion point 3. And then I would also mention that in the abstract

AliciaTak replied:
3. Consider adding an example of what's essential to consider from an end-user point of view. For example, ACS aims to publish bias-corrected data on a 5km grid; this decision is mainly based on user needs who want to have higher and higher resolved data. But I am not sure if that's an example for data creation rather than publish procedure.

paolap replied:
I've made this changes pushing now, I also change the abstract part to a more comprehensive  "dataset web page" to include keywords, related works , tile etc.

paolap replied:
sorry Alice I missed your comment before, yes probably that's nearly going into data product while here is more about what to choose among what's available, rather than making a decision at the start of the project on what the output should look like.

paolap commented on Governance/publish/publish-procedure.md:
```diff
@@ -0,0 +1,97 @@
+
+**Abstract**
+
+This ...
```
That's the part I didn't have yet time to write :-) I will add it now that I'm a bit more confident I was going in the right direction

paolap replied:
I Worte that now, see comment above

AliciaTak commented on Governance/publish/publish-procedure.md:
```diff
@@ -0,0 +1,97 @@
+It's not usual or a good thing to publish all the data produced in a research project. It's important to identify what data is useful or required, considered other limitations like storage availability. Here are some considerations that can help with the choice.
+1. Provide the information needed to interpret, reuse and reproduce the results. This is what journal publishers usually require, most of them provides [guidelines and examples](journals) of which data should be shared.
+ 
+2. If the output is big publish only a subset. If methods are well described, the software used is easily available, then publishing only the subset of data that underlines a publication is sufficient. For example, the post-processed output is sufficient for a model simulation. However, the model version and configuration used, the input data and model source code should be clearly documented.
```
comma after "big"

AliciaTak commented on Governance/publish/publish-procedure.md:
```diff
@@ -0,0 +1,97 @@
+The way files are organised in folders, their names and sizes should consider both how the files will be distributed and how they will be used. For example, the protocol used to download the files might have a maximum size allowed, likewise having to list a lot of small files it's inefficient when loading an html page. Names also should be descriptive enough that a file can be recognised easily a ...
+
+**Conventions**
+It is important to use [conventions](../concepts/conventions) and [controlled vocabularies](../concepts/controlled-vocab) whenever possible, both official ones, like CF conventions for file attributes, and others which are not a requirement but have become common practice in the climate community (e.g. CMIP variable names). As some of these conventions also apply to folder and file names, it is i ...
```
You could mention the CORDEX convention as an example, too. One thing I learned recently is that the CORDEX convention keeps changing (aka updating), and if one is not aware of the changing convention happening every couple of weeks, one might miss updating the data file and not be CORDEX aligned anymore. Currently, there are data description and meta-data differences between ACS-CCAM and QLD-CCAM published CMIP6 CORDEX data because of that.

paolap replied:
I was trying to give some generic advice, hence I mentioned only CF, we should have a publishing with ESGF where all these quirks can be added. CMOR, CORDEX etc. I will copy this comment to the relevant issue. And also could be added to other-conventions file? We should be covering cortex there

paolap replied:
see issue #28

paolap commented:
Uhh this is confusing I think I answered some of Alicia comments in the files list, can't see my answers here, anyway hopefully they will appear, and I think I covered all the reviews, but if I missed something let me know, I added the "abstract" now renamed "dataset web page" that was fully missing and also re-formatted the readme example in tech. There might be more pages like that that need reviewing. I will check tomorrow while we see if Chloe wants to also review or is happy with the changes.
One thing I noticed is that we maybe should remove the page : "Preparing files for publication" from create section. This hasn't been really populated yet and the publishing procedure should at least cover some of that. Some of the topics listed are specific to ESGF and could be covered in the publish with ESGF page that also we still need to fill.

hot007 commented:
Thanks good to go from me :)
